### PR TITLE
Add missing documentation changes to 2201.0.x branch

### PR DESCRIPTION
**Pull request by sachiniSam**
## Purpose
This pr adds the missing documentation commits which are currently present in the master

related issue https://github.com/wso2-enterprise/internal-support-ballerina/issues/110

Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/symbol/SymbolDocumentation.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+package org.ballerinalang.langserver.extensions.ballerina.symbol;
+
+import io.ballerina.compiler.api.symbols.Documentation;
+
+import java.util.List;
+
+/**
+ * Represents the documentation structure of getSymbol endpoint of BallerinaSymbolService.
+ *
+ *  @since 2201.1.0
```
Comment (malinthar) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/symbol/SymbolDocumentation.java:25]: let's change the version

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/symbol/SymbolInfoRequest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ * limitations under the License.
+ */
+package org.ballerinalang.langserver.extensions.ballerina.symbol;
+
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.TextDocumentIdentifier;
+/**
+ * Represents the request for getSymbol endpoint of BallerinaSymbolService.
+ *
+ * @since 2201.1.0
```
Comment (malinthar) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/symbol/SymbolInfoRequest.java:23]: version

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/symbol/SymbolInfoResponse.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.langserver.extensions.ballerina.symbol;
+
+import io.ballerina.compiler.api.symbols.SymbolKind;
+/**
+ * Represents the response for getSymbol endpoint of BallerinaSymbolService.
+ *
+ * @since 2201.1.0
```
Comment (malinthar) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/symbol/SymbolInfoResponse.java:22]: version

**Review by malinthar — COMMENTED** [review]
Changes look good! Lets address the following minor comments

**Review by tharindulak — APPROVED** [review]

**Pull request merged**